tests/jvm/src/test/resources/example/Classes.scala

```diff
@@ -23,11 +23,58 @@ class C8/*<=classes.C8#*/(private[this] val x/*<=classes.C8#x.*/: Int/*=>scala.I
 class C9/*<=classes.C9#*/(private[this] var x/*<=classes.C9#x().*/: Int/*=>scala.Int#*/)
 
 object N/*<=classes.N.*/ {
-  val anonClass/*<=classes.N.anonClass.*/ = new C7/*=>classes.C7#*/(42) {
+  val anonClass/*<=classes.N.anonClass.*/ = new /*<=local8*/C7/*=>classes.C7#*/(42) {
     val local/*<=local9*/ = ???/*=>scala.Predef.`???`().*/
   }
   val anonFun/*<=classes.N.anonFun.*/ = List/*=>scala.collection.immutable.List.*/(1).map/*=>scala.collection.immutable.List#map().*/ { i/*<=local10*/ =>
     val local/*<=local11*/ = 2
     local/*=>local11*/ +/*=>scala.Int#`+`(+4).*/ 2
   }
 }
+
+object CaseClasses1/*<=classes.CaseClasses1.*/ {
```

Comment:
I think we can remove the case classes here and just leave the chain, since the issues was with chained methods and params, not really with case classes.

Reply:
Removed.